pom.xml

```diff
@@ -9,36 +9,6 @@
     <version>1.0-SNAPSHOT</version>
     <packaging>jar</packaging>
 
-    <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot -->
-    <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.3.RELEASE</version>
-    </parent>
-    <properties>
```

Comment:
Properties section is ok - you say to maven that you're using java in 1.8 version which is true.